src/gsy_e/gsy_e_core/sim_results/endpoint_buffer.py

```diff
@@ -217,7 +218,7 @@ def _read_forward_markets_stats_to_dict(
 
         for market_type, market in area.forward_markets.items():
             for time_slot in market.market_time_slots:
-                time_slot_str = time_slot.format(DATE_TIME_FORMAT)
+                time_slot_str = datetime_to_string_incl_seconds(time_slot)
```

Comment:
Lets stick to using pendulum on the gsy-e side and use the DATE_TIME_FORMAT_SECONDS format:
```suggestion
                time_slot_str = time_slot.format(DATE_TIME_FORMAT_SECONDS)
```

Reply:
Why are the seconds needed BTW? Is the a requirement of the B2B?

Reply:
The format for time_slot here and in the Bid/Offer/Trade object should be the same. Otherwise my [assertions](https://github.com/gridsingularity/gsy-framework/blob/86001fcfe3cc8862f8f9c2c57c50639e12d5c2cf/gsy_framework/sim_results/electric_blue/aggregate_results.py#L80), fail. I could have written my assertions in some other way to get along with this difference, but the cost is higher that way.

Reply:
This has to be decided collectively, because until now, a time_slot_str never included the seconds because we did not see the time when there will be time_slots that are shorter than one minute. 
This is why I vote for leaving the seconds out, even if it means some more work for you, sorry.

Reply:
There's no problem on my side. I just wanted to avoid unnecessary conversions on gsy-framework.